tpgtools/product.go

```diff
@@ -93,6 +90,31 @@ func (pm *ProductMetadata) ProductBasePathDetails() *ProductBasePathDetails {
 	return &bp
 }
 
+func getProductTitle(documentTitle, packagePath string) string {
```

Comment:
nit: While not strictly required, I think describing the rules of the function in a header comment will make it easier to understand in the future.